automation/taskcluster/decision_task.py

```diff
@@ -74,7 +74,30 @@ def pr_or_push(is_push):
     ):
         other_tasks[taskcluster.slugId()] = craft_function()
 
-    if is_push and SHORT_HEAD_BRANCH == 'master':
+    return (build_tasks, signing_tasks, other_tasks)
+
+
+def push():
```

Comment:
Nit: I wonder if we should apply the same logic as the one implemented in r-b https://github.com/mozilla-mobile/reference-browser/pull/790/files#diff-168d551d2c0830884a5caeb50ae576e9R79

Reply:
@JohanLorenzo done!  thanks Johan